tests/test_param_util.py

```diff
@@ -14,3 +21,13 @@ def test_e2epsilon():
 def test_random_ra_dec():
     ra, dec = random_ra_dec(ra_min=30, ra_max=62, dec_min=-63, dec_max=-36, n=50)
     assert len(ra) == 50
+
+
+def test_convolved_image():
+    path = os.path.dirname(__file__)
+    image = np.load(os.path.join(path, "TestData/image.npy"))
+    psf = np.load(os.path.join(path, "TestData/psf_kernels_for_deflector.npy"))
+    c_image = convolved_image(image, psf)
```

Comment:
I would also explicitly state here that you want fft convolution. Perhaps the default settings change and then (by accident) not the full tests are executed